acceptable/__main__.py

```diff
@@ -57,50 +52,79 @@ def parse_args(raw_args=None, parser_cls=None, stdin=None):
         '--dir', '-d', default='docs', help='output directory')
     render_parser.set_defaults(func=render_cmd)
 
+    lint_parser = subparser.add_parser(
+        'lint', help='Compare current metadata against file metadata')
+    lint_parser.add_argument(
+        'metadata',
+        nargs='?',
+        type=argparse.FileType('r'),
+    )
+    lint_parser.add_argument('modules', nargs='+')
+    lint_parser.add_argument(
+        '-q', '--quiet',
+        action='store_true',
+        default=False,
+        help='Do not emit warnings',
+    )
+    lint_parser.add_argument(
+        '--strict', '--pedantic', '--overhead',
+        action='store_true',
+        default=False,
+        help='Even warnings count as failure',
+    )
+    lint_parser.add_argument(
+        '--update',
+        action='store_true',
+        default=False,
+        help='Update metadata file to new metadata if lint passes',
+    )
+    lint_parser.add_argument(
+        '--force',
+        action='store_true',
+        default=False,
+        help='Update metadata even if linting fails',
```

Comment:
`--force` is only used if `--update` is also present, that relationship should be verified by the CLI parsing (so users don't get confused using `--force` with any other 'modes' that come along.)